packages/stylelint-plugin/README.md

```diff
@@ -32,7 +31,11 @@ These rules are included by default:
 - `kaizen/no-invalid-use-of-var-tokens-in-functions` - Detects invalid usages of Kaizen variable tokens within some functions that are known to be problematic such as `rgba`, `darken` or `mix`, and automatically fixes `rgba`, `rgb`, `add-alpha`, `transparentize` instances by ensuring the correct `-rgb-params` tokens are used.
 - `kaizen/no-invalid-use-of-var-tokens-in-equations` - Detects invalid usages of Kaizen tokens within equations. It won't be able to automatically fix many cases, but it can for some simple cases for example negating a Kaizen variable without a runtime `calc()` function.
 - `kaizen/all-token-imports-must-be-used` - Reports and fixes unused Kaizen token imports.
-- `kaizen/tokens-must-be-imported` - Reports and fixes any Kaizen token imports that should exist but don't, based on the tokens that are used in the current stylesheet.
+- `kaizen/all-used-tokens-must-be-imported` - Reports and fixes any Kaizen token imports that should exist but don't, based on the tokens that are used in the current stylesheet.
```

Comment:
Renaming a rule is a breaking change - someone may be overriding this rule in particular (either ignoring it completely or changing the default log to error) and this would break their pipeline.

Reply:
Hey sorry i wasn't trying to rename it, i was trying to fix the README because currently the name of the rule is actually `kaizen/all-used-tokens-must-be-imported` as per the file `packages/stylelint-plugin/src/rules/all-used-tokens-must-be-imported.ts`

Reply:
Oh - my mistake. Carry on then 😬